### PR TITLE
[SP-6351] - Backport of PDI-19626 - The Repository Manager in Spoon will not display all fields/buttons with certain resolution settings (9.3 Suite)

### PR DESCRIPTION
**Pull request by bcostahitachivantara**
@smmribeiro 
@andreramos89 
@renato-s

### CODE REVIEW

File: plugins/repositories/core/src/main/javascript/app/components/database/select/select.html [hunk 1]
```diff
@@ -1,5 +1,90 @@
...
+              id="repository-manager-database-select-buttons-btnBack"
+              class="btn btn-color-primary"
+              ng-click="vm.back()">
+        <span> {{::vm.backLabel}} </span>
+      </button>
+    </div>
+  </div>
+</div>
+
+<!--
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/database/select/select.html:85]: Is this commented section important?

File: plugins/repositories/core/src/main/javascript/app/components/file/details/details.html [hunk 2]
```diff
@@ -1,5 +1,89 @@
...
+              id="repository-manager-file-details-row-action-buttons-btnBack"
+              class="btn btn-outline-color-secondary"
+              ng-click="vm.back()">
+        <span> {{::vm.backLabel}} </span>
+      </button>
+    </div>
+  </div>
+</div>
+
+<!--
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/file/details/details.html:84]: Is this commented section important?

File: plugins/repositories/core/src/main/javascript/app/components/database/details/details.css [hunk 3]
```diff
@@ -1,6 +1,87 @@
...
+  background-color: #0F2B5B;
+  border-color: #0F2B5B;
+}
+
+.form-control:focus {
+  border-color: #005EAA;
+  box-shadow: 0 0 0 0.2rem rgba(0,94,170,.25);
+}
+
+#repository-manager-database-details-header {
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/database/details/details.css:34]: Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.

File: plugins/repositories/core/src/main/javascript/app/components/add/add.css [hunk 4]
```diff
@@ -1,12 +1,67 @@
...
+  border: 1px solid #005DA6;
+}
+
+.btn-outline-color-secondary:hover {
+  color: #0F2B5B;
+  background-color: #FFFFFF;
+  border: 1px solid #0F2B5B;
+}
+
+#repository-manager-add-container {
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/add/add.css:34]: Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.

File: plugins/repositories/core/src/main/javascript/app/components/database/select/select.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    border: 1px solid #005DA6;
+}
+
+.btn-outline-color-secondary:hover {
+    color: #0F2B5B;
+    background-color: #FFFFFF;
+    border: 1px solid #0F2B5B;
+}
+
+#repository-manager-database-select-row-list-empty-info {
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/database/select/select.css:34]: Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.

File: plugins/repositories/core/src/main/javascript/app/components/file/details/details.css [hunk 6]
```diff
@@ -1,6 +1,59 @@
...
+  background-color: #0F2B5B;
+  border-color: #0F2B5B;
+}
+
+.form-control:focus {
+  border-color: #005EAA;
+  box-shadow: 0 0 0 0.2rem rgba(0,94,170,.25);
+}
+
+#repository-manager-file-details-header {
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/file/details/details.css:34]: Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.

File: plugins/repositories/core/src/main/javascript/app/components/manager/manager.css [hunk 7]
```diff
@@ -1,3 +1,72 @@
...
+
+.repository-manager-button-hidden {
+  display: none;
+}
+
+#repository-manager-row-action-buttons > div {
+  padding: 4px;
+}
+
+/* The following css need to be validate which CSS can be remove or not and which css should be migrated into global.css */
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/manager/manager.css:69]: Is this for now or just a reminder for a future change?

File: plugins/repositories/core/src/main/javascript/app/components/other/other.css [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    border: 1px solid #005DA6;
+}
+
+.btn-outline-color-secondary:hover {
+    color: #0F2B5B;
+    background-color: #FFFFFF;
+    border: 1px solid #0F2B5B;
+}
+
+#repository-manager-other-table-wrapper {
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/other/other.css:34]: Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.

File: plugins/repositories/core/src/main/javascript/app/components/pentaho/details/details.css [hunk 9]
```diff
@@ -1,12 +1,53 @@
...
+  background-color: #0F2B5B;
+  border-color: #0F2B5B;
+}
+
+.form-control:focus {
+  border-color: #005EAA;
+  box-shadow: 0 0 0 0.2rem rgba(0,94,170,.25);
+}
+
+#repository-manager-details-header {
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/pentaho/details/details.css:28]: Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.

File: plugins/repositories/core/src/main/javascript/app/components/shared/failure/failure.css [hunk 10]
```diff
@@ -1,6 +1,69 @@
...
-  height: 58px;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 65px;
+  height: 60px;
   margin: auto;
+  padding: 50px;
 }
+
+#repository-manager-failure-header {
```
Comment (smmribeiro) [inline plugins/repositories/core/src/main/javascript/app/components/shared/failure/failure.css:11]: Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.

File: plugins/repositories/core/src/main/javascript/app/components/file/details/details.html (hunk 2 above)
Comment (bcostahitachivantara) [reply plugins/repositories/core/src/main/javascript/app/components/file/details/details.html:84]: i forgot to remove it, thank you

**Pull request merged**